pkg/kn/commands/revision/list.go

```diff
@@ -223,7 +224,7 @@ func enrichRevisionAnnotationsWithServiceData(serviceFactory serviceFactoryFunc,
 }
 
 // Create a function for being able to lookup a service for an arbitrary namespace
-func serviceLookup(serviceFactory serviceFactoryFunc) serviceGetFunc {
+func serviceLookup(serviceFactory serviceFactoryFunc, ctx context.Context) serviceGetFunc {
```

Comment:
Shouldn't be `Context` param first?
```suggestion
func serviceLookup(ctx context.Context, serviceFactory serviceFactoryFunc) serviceGetFunc {
```

Reply:
Good catch.